backend/app/assets/tailwind/application.css

```
@@ -1 +1,79 @@
 @import "tailwindcss";
+
+@theme {
+  /*-------color-------*/
+  --color-primary: #1A365D;
+  --color-accent: #C09357;
+  --color-background: #FDFBF7;
+  --color-surface: #FFFFFF;
+
+  /*-------font-------*/
```

Stylelint (17.10.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)